addons/stock/models/stock_picking.py

```diff
@@ -87,6 +87,13 @@ def _default_show_operations(self):
     company_id = fields.Many2one(
         'res.company', 'Company', required=True,
         default=lambda s: s.env.company.id, index=True)
+    create_backorder = fields.Selection(
+        [('ask', 'Ask'), ('always', 'Always'), ('never', 'Never')],
+        'Create Backorder', required=True, default='ask',
+        help="When validating a transfer:\n"
+             " * Ask: user is asked to choose if he wants to make a backorder for remaining products\n"
```

Comment:
```suggestion
             " * Ask: user is asked to choose if they want to make a backorder for remaining products\n"
```
I think it's better to be gender neutral instead of assuming users are always men :grimacing: But not sure about the conjugation :thinking:

Reply:
Haha, correct